### DISCUSSION TITLE
How do you install your Zigbee Stick?

### DISCUSSION THREAD

**Discussion by Danit2** [Koenkk/zigbee2mqtt#9839] [Show and tell]
This discussion is meant to show how people got their Zigbee Stick installed.

I'll be happy to start.

I have install my Stick in the living room in the wall. 
![image](https://user-images.githubusercontent.com/71522810/143446868-3e1e8482-e509-4fb9-800f-188b7f3923ff.jpeg)

It is a SMARTLIGHT CC2652P. 
 
In the wall i have a usb extension cable and this go to my Raspberry Pi 4. 
![image](https://user-images.githubusercontent.com/71522810/143447355-1836885e-ee09-4a1d-a14f-4e4cdb240017.jpeg)

What do you think and how have you do that?

**Comment by sergge1** [comment]
You can use LAN Zigbee adapter and install it wherever you want - have a look there https://smartlight.me/smart-home-devices/zigbee-devices/smlight-zigbee-lan-adapter-slzb-05en
